[Integration][ADO] Fix missing team context for non default teams while ingesting boards (#1127)
port-labs/ocean: integrations/azure-devops/CHANGELOG.md

<!-- towncrier release notes start -->

## 0.1.79 (2024-11-07)

### Bug Fixes

> - Fixed Azure DevOps Boards integration to include team context, API endpoint will iterate through all project teams, ensuring non-default team boards and columns are ingested

Reviewer: ```suggestion
- Fixed the API endpoint used in the boards kind to iterate through all project teams, ensuring non-default team boards and columns are ingested
```
